### PR TITLE
S5T1: Service for transfer data from parsers to DB

### PR DESCRIPTION
**Pull request by greySlon**
here is ForecastCollector that collect necessary data to create the forcasts from all  of the parsers, sends them further for user notification and rewrites forecasts in tables in database. It will be started from some REST service.

### CODE REVIEW

File: src/main/java/com/cityelf/utils/ForecastCollector.java [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+        for (Address address : addresses) {
+          WaterForecast forecast = new WaterForecast();
+          forecast.setStart(forcastData.getStartOff());
+          forecast.setEstimatedStop(forcastData.getEndOff());
+          forecast.setAddress(address);
+          forecasts.add(forecast);
+        }
+      }
+    } catch (ParserUnavailableException ex) {
+      ex.printStackTrace();
```
Comment (parkhomenko) [inline src/main/java/com/cityelf/utils/ForecastCollector.java]: Let's add a logger, so the error messages are written in some file

**Review by parkhomenko — CHANGES_REQUESTED** [review]
Everything is great. Just add a controller with a single method which starts your forecast collector. The mapping could be something like "/forecasts/collector/start".

**Comment by greySlon** [comment]
Added controller and logging in ForecastCollector(console out only)
Tried to get output in file but there were some difficulties - standart logger configuration worked and my configuration never applied

**Review by parkhomenko — APPROVED** [review]

**Comment by parkhomenko** [comment]
We can skip logging in file for now.

**Pull request merged**